packages/analytics-core/src/timeline.ts

```diff
@@ -95,11 +96,11 @@ export const apply = async () => {
 };
 
 export const flush = async (config: Config) => {
-  const destination = config.plugins.filter<DestinationPlugin>(
-    (plugin: Plugin): plugin is DestinationPlugin => plugin.type === PluginType.DESTINATION,
+  const destination = config.plugins.filter<AmplitudeDestinationPlugin>(
+    (plugin: Plugin): plugin is AmplitudeDestinationPlugin => plugin.type === PluginType.DESTINATION,
```

Comment:
Feels like a hack/workaround to not use `@ts-ignore` 😅 But it's okay for now. Let me see if there are other ways

Reply:
Sounds good.